src/tool/natvis/pch.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #define WIN32_LEAN_AND_MEAN             // Exclude rarely-used stuff from Windows headers
+#define NOMINMAX
```

Comment:
Isn't this specified globally in the root cmake file?

Reply:
The msbuild project doesn't pick up the cmake defaults.

Reply:
correct

Reply:
👍